unittests/linuxliketests.py

```diff
@@ -54,6 +56,18 @@
 from .baseplatformtests import BasePlatformTests
 from .helpers import *
 
+def _prepend_pkg_config_path(path: str) -> str:
```

Comment:
Feels like there must be a helper function for this in the source code already that this guy could make use of

Reply:
Not that I'm aware of. Usually this is just `os.path.pathsep.join(list_of_paths)`